src/modules/ProductDetailsPage/components/ColorCapacityComponent/ColorCapacityComponent.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import cn from 'classnames';
+import styles from './ColorCapacityComponent.module.scss';
+import { PhoneDetail } from '../../../../types/PhoneDetail';
+import { ReactComponent as Color }
+  from '../../../../static/icons/color-icon.svg';
+import { useAppSelector } from '../../../../store/hooks';
+
+type Props = {
+  phoneDetails: PhoneDetail | null,
+  setColor: (value: string) => void,
+  setCapacity: (value: string) => void,
+};
+
+export const ColorCapacityComponent: React.FC<Props> = ({
+  phoneDetails,
+  setColor,
+  setCapacity,
+}) => {
+  const { isDarkTheme } = useAppSelector((state) => state.theme);
+
+  return (phoneDetails
+    && (
+      <section
+        className={cn(styles.colorCapacity, {
+          [styles.colorCapacity__DARK]: isDarkTheme,
+        })}
+      >
+        <div className={styles.contentContainer}>
+
+          <p
+            className={cn(styles.content, {
+              [styles.content__DARK]: isDarkTheme,
+            })}
+          >
+            Available colors
+          </p>
+
+          <div className={styles.buttonContainer}>
+            {isDarkTheme
+              ? (
+                phoneDetails.colorsAvailable.map(color => (
+                  // eslint-disable-next-line jsx-a11y/control-has-associated-label
```

Comment:
I'm not sure, but since you have <Color> inside, maybe the linter will not consider this as a problem now?